docs/Integration/alluxio-guide.md

```diff
@@ -0,0 +1,44 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more 
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership. 
+    The ASF licenses this file to you under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with 
+    the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software 
+    distributed under the License is distributed on an "AS IS" BASIS, 
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and 
+    limitations under the License.
+-->
+
+
+# Presto guide
+This tutorial provides a quick introduction to using Alluxio.
+
+## How to use Alluxio for CarbonData?
+### Install and start Alluxio
+Please refer to [https://www.alluxio.org/docs/1.8/en/Getting-Started.html#starting-alluxio](https://www.alluxio.org/docs/1.8/en/Getting-Started.html#starting-alluxio)   
+Access the Alluxio web: [http://localhost:19999/home](http://localhost:19999/home)   
+By command, for example: 
+```$xslt
+./bin/alluxio fs ls /
+```
+Result:
+```
+drwxr-xr-x xubo           staff                        1   NOT_PERSISTED 01-07-2019 15:39:24:960  DIR /carbondata
+-rw-r--r-- xubo           staff                    50686   NOT_PERSISTED 01-07-2019 11:37:48:924 100% /data.csv
+```
+### Upload Alluxio jar to CarbonData
+Upload the jar "/alluxio_path/client/alluxio-YOUR-VERSION-client.jar" to CarbonData
```

Comment:
"Upload to CarbonData" is confusing. What we need to do is to add the alluxio client jar to classpath, right?

Reply:
No need for example now, I added dependency